client/src/js/BrokenExternalLinksReport.js

```diff
@@ -103,6 +112,10 @@
 
             // If complete status
             if (data.Status === 'Completed') {
+              if (self.getReloadContent()) {
+                $('.cms-container').loadPanel(document.location.href, null, {}, true, false);
+                self.setReloadContent(false);
```

Comment:
I do this just in case. Since this value will return to false anyway after the content is updated.